Source/Core/AudioCommon/Mixer.cpp

```diff
@@ -282,8 +292,10 @@ void Mixer::PushStreamingSamples(const short* samples, unsigned int num_samples)
 void Mixer::PushWiimoteSpeakerSamples(const short* samples, unsigned int num_samples,
                                       unsigned int sample_rate_divisor)
 {
-  short samples_stereo[MAX_SAMPLES * 2];
+  static std::array<short, MAX_SAMPLES * 2> samples_stereo;
```

Comment:
`static`?

Reply:
We don't really need to allocate it on the stack, but I guess there's no harm in doing so either. I don't remember intentionally changing it to `static` for what it's worth.

Also, on further unrelated investigation, it's not possible for there to be more than 40 wiimote samples at a time (due to the maximum size of the output report combined with 4-bit ADPCM). So this array is probably way larger than it really needs to be.

Reply:
I'd rather you leave this on the stack, especially when it's much smaller anyway (maybe make that change and add an assert?), we have enough hard-to-find global state as-is...